### ISSUE TITLE
Convert safe-rest to use https even in development

### ISSUE THREAD

**Issue by sdurbin1** [sdurbin1/safe-rest#40]
Currently, we only use https in production mode.  Long term, we will also want to use https in development (just not for this two week deadline)

**Comment by jeffshaver** [comment]
isn't this just a point of swapping out the `http` and `https` module?

**Comment by sdurbin1** [comment]
Yep, but the reason I wanted to hold off, is the effort of getting everyone set up with using fake PKI certs.  Don't want to break everyone's development environments at the moment... (for little benefit)
